cpp/harvest/marc_issn_lookup.cc

```diff
@@ -223,8 +304,9 @@ int Main(int argc, char **argv) {
     if (argc != 4)
         Usage();
 
-    std::map<std::string, SubFieldInfo> journal_cache(BuildJournalCache(argv[2]));
+    std::vector<SubFieldInfo> journal_cache(BuildJournalCache(argv[2]));
     ISSNLookup(argv, journal_cache);
 
+    PrettyPrintCache(journal_cache);
```

Comment:
Maybe we could add an optional flag `[--verbose]` in the usage and only print the cache if it is set.